### ISSUE TITLE
Missing tooltips from certain tldraw tools

### ISSUE THREAD

**Issue by antobinary** [bigbluebutton/bigbluebutton#16147]
**Describe the bug**
Both for accessibility context and for pure informative reasons. The conrols that have a red arrow point to them in the below image are missing tooltips.

![Screenshot from 2022-12-15 12-12-45](https://user-images.githubusercontent.com/6312397/207925254-b241b708-6f76-4305-9790-0669f346b64f.png)

**Comment by antobinary** [comment]
Likely resolved via https://github.com/tldraw/tldraw/pull/1106

We can close this issue after pulling the latest tldraw (including the above PR) and retesting

**Comment by KDSBrowne** [comment]
The tooltips for the undo/redo was included in this PR https://github.com/tldraw/tldraw/pull/1095

**Comment by proke03** [comment]
Tooltips for Shapes button are included in the menu.
![캡처_2023_01_12_20_12_50_525](https://user-images.githubusercontent.com/19148682/212052290-a282d046-112b-4e49-ac17-d950ea942dc8.png)

**Comment by proke03** [comment]
I think that there are no shorcut for style buttons yet.
